tools/uwuw_preproc

```diff
@@ -370,56 +401,61 @@ def copy_metadata(material, material_from_lib):
 
 
 """
-function to set the attributes of the materials:
+function to set the metadata of materials:
 ----------------------------------------
 mat : PyNE Material Object
 number : mcnp material number
-flukamat_list : 
+flukamat_list : list of materials  
 returns : PyNE Material Object
 """
+
+
 def set_metadata(mat, number, flukamat_list):
     mat.metadata['mat_number'] = str(number)
-    mat.metadata['fluka_material_index'] = str(number+25)
+    mat.metadata['fluka_material_index'] = str(number + 25)
```

Comment:
@moatazharb I think you need to talk to @makeclean about this and find out where this number is coming from. Numbers like this really need to be documented to avoid:

https://en.wikipedia.org/wiki/Cargo_cult_programming